app/src/scripts/utils/bids.js

```diff
@@ -438,7 +456,7 @@ export default {
     }
     dataset.stars = this.stars(dataset, stars)
     dataset.starCount = dataset.stars ? '' + dataset.stars.length : '0'
-
+    dataset.followers = '' + this.followers(dataset, followers).length || '0'
```

Comment:
the if else shorthand is like `conditional ? a : b`, as opposed to `conditional ? a || b`